src/plugins/stake-plugins/thorchainSavers/tcSaversPlugin.ts

```diff
@@ -414,7 +424,21 @@ const stakeRequest = async (opts: EdgeGuiPluginOptions, request: ChangeQuoteRequ
     }
   }
 
-  const totalFee = add(slippageNativeAmount, needsFundingPrimary ? mul(networkFee, '2') : networkFee)
+  const fee = needsFundingPrimary ? mul(networkFee, '2') : networkFee
+  const futureUnstakeFee = await futureUnstakePromise
+
+  // Calculate the amount of time needed to break even from just fees
+  const totalFee = add(add(fee, slippageNativeAmount), futureUnstakeFee)
+  const policy = policies.find(policy => policy.stakePolicyId === stakePolicyId)
+  if (policy == null) {
+    throw new Error(`Cannot find policy ${stakePolicyId}`)
+  }
+  const totalFeePercent = (Number(totalFee) / Number(nativeAmount)) * 100
+  const { apy } = policy
+
+  const breakEvenYears = totalFeePercent / apy
+  const breakEvenDays = breakEvenYears * 365
```

Comment:
Why did you choose to not use big numbers in this math?

Reply:
Won't really lose precision given we are rounding and truncating anyway